constants/error-constants.js

```diff
@@ -82,6 +82,7 @@ const ERROR = {
     INVALID_VALIDATE_METADATA: "You do not have permission to validate the metadata",
     INVALID_PERMISSION_TO_VIEW_VALIDATION_RESULTS: "You do not have permission to view the validation results for this submission",
     INVALID_PERMISSION_TO_VIEW_NODE_TYPES: "You do not have permission to view the node types included in this submission",
+    MISSING_SUBMISSION_FILE_ERRORS: "Submission's file errors is missing in the submission",
```

Comment:
Can we change this to "The file errors property is missing from the submission

Reply:
Applied the requested change.